sw/device/sca/lib/aes.h

```diff
@@ -0,0 +1,194 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SCA_LIB_AES_H_
+#define OPENTITAN_SW_DEVICE_SCA_LIB_AES_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/memory.h"
+
+/**
```

Comment:
IIUC, only `aes_encrypt_block()`, `aes_encrypt_block()` and maybe the transpose functions should be calleable. This means all other functions could be declared inside `aes.c` directly and use the `static` key word. A SW expert may provide some clearer guidance here.

Reply:
I agree.
This was a leftover from the debugging.
Only `aes_sw_encrypt_block` and `aes_key_schedule` are now in `aes.h`

Reply:
Nice, thanks!